app/views/shared/_title.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1 class="page-header"><%= document.title %>
+  <ul class='document-slug'>
+    <li title='<% if !document.published? %>When published this document will appear at this URL<% end %>'>
```

Comment:
No, a document is draft once it's been created. When a document has been published and a new draft created, both `document.published?` and `document.draft?` are true.